packages/components/src/border-control/border-control-dropdown/component.tsx

```diff
@@ -247,9 +242,7 @@ const BorderControlDropdown = (
 		<Dropdown
 			renderToggle={ renderToggle }
 			renderContent={ renderContent }
-			popoverProps={ {
-				...__unstablePopoverProps,
-			} }
+			popoverProps={ __unstablePopoverProps }
```

Comment:
No, I simply wanted to simplify the code and avoid the creation of a new object every time — was that intended? Happy to revert

Reply:
Seems intentional to me, although it's unclear whether it's warranted. If you haven't looked into it specifically, it might be better to bump that change to a separate issue, since it seems like the kind of thing that would have *implications* 👻 (It might be good implications though!)

Reply:
The last PR to edit those lines was #41900

Anyway, I went ahead and reverted the above change in https://github.com/WordPress/gutenberg/pull/43823/commits/dac81ea2a8b33f5e9d28d17a0491db8c70adb93c